app/Language/en/Foobar.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Config\App;
```

Comment:
I am not sure why you have these two test language files in your PR. And causes the failing tests. This is part of the Generator commands test and should in no way end up here. Is your fork not updated?